scripts/create_baseline_stubs.py

```diff
@@ -79,9 +127,19 @@ def create_metadata(stub_dir: str, version: str) -> None:
     version = match.group(0)
     if os.path.exists(filename):
         return
+    metadata = f'version = "{version}.*"'
+    upstream_repo_url = asyncio.run(get_upstream_repo_url(project))
+    if upstream_repo_url is None:
+        warning = (
+            f"\nDid not succeed in finding a URL pointing to the upstream repo for {project!r}.\n"
+            f"Please add it manually to the `stubs/{project}/METADATA.toml` file, if possible!\n"
```

Comment:
It could be hard to tell how exactly you're supposed to add it to the file. I think this is enough, if we assume the contributor looks at `CONTRIBUTING.md` or other `METADATA.toml` files:
```suggestion
            f"Please add it as `upstream_repository` to `stubs/{project}/METADATA.toml`, if possible!\n"
```